specification/personal-demographics.yaml

```diff
@@ -928,7 +928,7 @@ paths:
 
         * the `id` - the object ID for items that have one - for example `name`, `address` or `telecom`.
         * the `url` - the URL for the extension being removed.
-        * the whole object being removed.
+        * the whole object being removed. (this is the only allowed option for patient access updates)
```

Comment:
@PeterGallagherBJSS , have this documentation changes been captured in the Jira ticket? We should get the wording reviewed by Jack

Reply:
No I should've though. I'll ask him now.

Reply:
Resolving as Jack is happy with the doc change